srcpkgs/autotiling/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'autotiling'
+pkgname=autotiling
+version=1.4
+revision=1
+build_style=python3-module
+makedepends="python3-pip"
```

Comment:
Don't use `pip` in template

Reply:
`hostmakedepends="python3-setuptools"` instead

Reply:
Thanks for the super quick check! :)
I was wondering why it was fine to build locally, though are you sure setuptools is required? I'm under the impression from the manual that it's implied by the build style.

Reply:
`python3-wheel` is redundant.
You can add `checkdepends="python3-pytest"` to run test

Reply:
`python3-wheel` does seem to be necessary, as it's specified as a [setup dependency](https://github.com/nwg-piotr/autotiling/blob/master/setup.cfg). I'll try with your other suggestions. :)

Reply:
My bad on the suggestions :)
I believe `python3-setuptools` already has its `wheel`.
```
$ xbps-query -f python3-setuptools | grep wheel
/usr/lib/python3.9/site-packages/setuptools/wheel.py
```
I checked the repo again. There is no test. So you can skip the `do_check()` by adding `make_check=no` below the checksum

Reply:
Thanks a bunch for your help! It seems to be working now.